train_model.py

```diff
@@ -369,27 +213,47 @@ def train_model(
 
     # Print estimated loss if it hasn't learned anything
     print("\nEstimated Loss if guessing:")
-    print(f"-log(1 / {vocab_size}) = {-torch.log(torch.tensor(1 / vocab_size))}")
+    print(f"-log(1 / {args.vocab_size}) = " + \
+        f"{-torch.log(torch.tensor(1 / args.vocab_size))}")
+    
+    # Get Tokenizer from local directory
+    tokenizer = PreTrainedTokenizerFast.from_pretrained(args.tokenizer_folder)
+
+    # Loads Tokenized data
+    tokenized_dataset = load_ds(
+        "parquet",
+        data_files=str(Path(args.dataset_dir) / args.dataset_name / f"{args.dataset_subset}.parquet"),
+        split="all")
+
+    train_loader = DataLoader(
+        tokenized_dataset["train"].with_format("torch")["input_ids"],
+        batch_size=args.batch_size,
+        shuffle=True)
```

Comment:
Can you double check that shuffle does not need something like a random seed to ensure repeatability in tests?